server/src/main/java/io/spine/server/entity/storage/Methods.java

```diff
@@ -44,13 +44,25 @@
 
 /**
  * Utilities for working with methods.
- *
- * @author Dmytro Dashenkov
- * @author Alexander Yevsyukov
  */
+@SuppressWarnings("DuplicateStringLiteralInspection") // Random duplication of simple strings.
 class Methods {
 
-    private static final String GETTER_PREFIX_REGEX = "(get)|(is)";
+    /**
+     * One of the possible getter prefixes - {@code get-}.
+     */
+    private static final String GET_PREFIX = "get";
+
+    /**
+     * The other possible getter prefix - {@code is-}.
+     *
+     * <p>Allowed for {@code boolean} and {@link Boolean} entity columns.
+     *
+     * <p>Package-private access because we manually search for some of such methods.
```

Comment:
At first glance this explanation doesn't really explain the reason. Can you please rephrase?